tdvt/TDVT Results.twb

```diff
@@ -59,17 +60,16 @@
       <connection class='federated'>
         <named-connections>
           <named-connection caption='test_results_combined' name='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf'>
-            <connection class='textscan' directory='/Users/tsjoblad/Desktop/new TDVT Workbook/For GitHub (1:11:21)' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />
+            <connection class='textscan' directory='/Users/s.davis/Desktop' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />

```

Comment:
Let's consider making this `directory=''` for better relative path error message. This workbook still requires the user to have test_results_combined.csv and test_metadata.csv in the same directory.